recipes/example/meta.yaml

```diff
@@ -1,83 +1,23 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
-# Using the name variable with the URL in line 13 is convenient
-# when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
+{% set version = "2.2.203" %}
+{% set sha256 = "d1e2368b1335a6a5f496b887950ad5da3d85783ec76f74a663214989817bb497" %}
 
 package:
-  name: {{ name|lower }}
+  name: dotnet
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  # If getting the source from GitHub remove the line above
-  # uncomment the line below and modify as needed
-  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
-  # sha256 is the prefered checksum -- you can get it for a file with:
-  #  `openssl sha256 <file name>`.
-  # You may need the openssl package, available on conda-forge
-  #  `conda install openssl -c conda-forge``
+  url: https://download.visualstudio.microsoft.com/download/pr/647f8505-3bf0-48c5-ac0f-3839be6816d7/d0c2762ded5a1ded3c79b1e495e43b7c/dotnet-sdk-{{ version }}-linux-x64.tar.gz
+  sha256: '{{ sha256 }}'
 
 build:
-  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
-  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
-  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
-  # noarch: python
+  skip: True # [win]
   number: 0
-  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
-  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
-  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
-  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
-  # Once resolved, it should be removed.
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-
-requirements:
-  build:
-    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
-    # compilers are named 'c', 'cxx' and 'fortran'.
-    - {{ compiler('c') }}
-  host:
-    - python
-    - pip
-  run:
-    - python
+  script: 
+    - tar xvf dotnet-sdk-{{ version }}-linux-x64.tar.gz
```

Comment:
e.g. on linux:
```
mkdir -p $PREFIX/dotnet && tar xvf dotnet-sdk-{{ version }}-{{ platform }}-x64.tar -C $PREFIX/dotnet
```
...though you'd want to have `$PREFIX/dotnet` on your path after activating the environment